frontend/src/index.tsx

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
+import 'bootstrap/dist/css/bootstrap.min.css';
```

Comment:
fixing bug the default class is placed after my custom class, so my css can't override the default class